### PR TITLE
group一覧の謎の枠線削除

### PR DESCRIPTION
**Pull request by tty007**
色々試しましたがこの方法しか効かなかったので、[このページ](https://teratail.com/questions/86704)を参考にして無理やり消しました。

### CODE REVIEW

File: app/javascript/groups.vue [hunk 1]
```diff
@@ -52,6 +52,9 @@ ul {
   list-style-type: none;
   padding: 0;
   margin: 0;
```
Comment (seiya0914) [inline app/javascript/groups.vue]: なるほど、結構無理やりですねｗ
でも、しょうがないと思うので、なぜこの実装にしたかコメントだけ残しておいてもらえるとー

Comment (tty007) [reply app/javascript/groups.vue]: 色々調べてこれに行き着きましたw
了解です！

**Comment by tty007** [comment]
コメント二箇所追記しました！不都合あればご指摘ください〜

File: app/javascript/groups.vue [hunk 2]
```diff
@@ -1,5 +1,6 @@
 <template>
   <div id="groups">
+    <Header />
```
Comment (seiya0914) [inline app/javascript/groups.vue]: @tty007 これuser-containerコンポーネントで一番外をラップしてあげる形にしてほしいですー

`<template>
    <user-container>
      <div class="container">
        
        <h1 class="groupName">{{ group.name }}</h1>
        
        <section class="groupMembers" v-if="group_members.length">
          <h2 class="sectionTitle">メンバー</h2>
          <swiper
            ref="memberThumnails"
            :options="swiperOption">
            <swiper-slide
              v-for="(member, key) in members"
              :key="key"
              class="memberThumnail">
              
            </swiper-slide>
            
            <swiper-slide v-if="members.length <= 4"/>
            <swiper-slide v-if="members.length <= 4"/>
            <swiper-slide v-if="members.length <= 4"/>
          </swiper>
          <div>
            <p class="memberNickName">{{ members[activeMemberThumnailIndex].nickname }}</p>
            <pre>{{ group_members[activeMemberThumnailIndex].comment }}</pre>
          </div>
        </section>
        <section class="groupDescription">
          <h2 class="sectionTitle">どんなグループ？</h2>
          <pre>{{ group.explanation }}</pre>
        </section>
        <section class="groupRegulation">
          <h2 class="sectionTitle">参加条件</h2>
          <pre>{{ group.regulation }}</pre>
        </section>
        <div class="footer" v-if="am_i_joining">
          <el-button
            type="primary"
            class="shadow"
            round>
            チャトる
          </el-button>
        </div>
        <div class="footer" v-else>
          <el-button 
            @click="dialogVisible = true"
            type="primary" 
            class="shadow"
            round>
            凸りたい
          </el-button>
        </div>
      </div>
      
      
      <el-dialog
        :visible.sync="dialogVisible"
        width="264px"
        style="margin-top: 15vh;">
        <el-form
          :action="`/groups/${group.id}/group_members`" 
          method="post"
          ref="join"
          style="width: 100%"
        >
          <authenticity-token :value="authenticity_token" />
          <el-input
            type="textarea"
            :rows="5"
            placeholder="グループの皆さんに向けてひとこと"
            style="font-size: 12px;"
            v-model="comment"
            name="comment" />
          </el-form>
        <span slot="footer">
          <el-button 
            type="primary" 
            @click="join"
            round
            style="margin-bottom: 20px;">
            凸る
          </el-button>
      </span>
      </el-dialog>
      
    </user-container>
</template>
<script>
import container from './components/user-container'
import authenticityToken from './components/authenticity-token-field'
import { swiper, swiperSlide } from 'vue-awesome-swiper'
import 'swiper/dist/css/swiper.css'

export default {
  data() {
    const self = this
    return {
      swiperOption: {
        slidesPerView: 4,
        slideToClickedSlide: true,
        on: {
          slideChangeTransitionStart() {
            self.activeMemberThumnailIndex = this.activeIndex
          }
        }
      },
      activeMemberThumnailIndex: 0,
      dialogVisible: false,
      comment: ''
    }
  },
  components: {
    'user-container': container,
    'swiper': swiper,
    'swiper-slide': swiperSlide,
    'authenticity-token': authenticityToken
  },
  props: ['group', 'members', 'group_members', 'am_i_joining', 'authenticity_token'],
  methods: {
    join() {
      this.dialogVisible = false;
      this.$refs.join.$el.submit()
    }
  }
}
</script>
<style lang="scss" scoped>
h1, h2 {
  font-weight: lighter;
}
//MEMO: アクティブなswiperのimgにボーダーをつける
.swiper-slide-active img {
  border: #606266 2px solid;
}
button {
  width: 100%;
}
.shadow {
  filter: drop-shadow(0 3px 6px rgba(0, 0, 0, 0.16));
}
.container {
  color: #707070;
  margin-bottom: 80px;
  pre {
    white-space: pre-wrap ;
    line-height: 1.5;
  }
  section {
    padding: 47px 0;
    margin: 0 18px;
  }
  .coverImage {
    height: 155px;
    width: 100%;
    object-fit: cover;
  }
  .sectionTitle {
    font-size: 20px;
  }
  .groupName {
    font-size: 19px;
    margin: 15px 8px 0;
  }
  .groupMembers {
    border-bottom: 1px solid #707070;
    .sectionTitle {
      margin-bottom: 13px;
    }
    .memberThumnail {
      position: relative;
      text-align: center;
      img {
        width: 70px;
        height: 70px;
        border-radius: 50%;
        margin: 0 5px;
      }
    }
    .memberNickName {
      padding: 24px 0;
    }
  }
  .groupDescription {
    border-bottom: 1px solid #707070;
    .sectionTitle {
      margin-bottom: 47px;
    }
  }
  .groupRegulation {
    .sectionTitle { 
      margin-bottom: 47px;
    }
  }
  .footer {
    left: 0;
    right: 0;
    width: 90%;
    margin: auto;
    display: flex;
    position: fixed;
    bottom: 40px;
  }
}
</style>
`

Comment (tty007) [reply app/javascript/groups.vue]: 遅くなってすいません、了解しました！

**Pull request merged**